### PR TITLE
Created new bookclub readme and updated it

### PR DESCRIPTION
**Pull request by j4sum**
## Type Of Change
- [ ] Update documentation
- [ ] Add new event details
- [x] Add project resources

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Commit message is as per guidelines in the contributor guide

### CODE REVIEW

File: resources/bookclub/README.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# **BOOK CLUB**
+
+The official book club repo for WWCode-London 📚
```
Comment (dricazenck) [inline resources/bookclub/README.md:3]:
```suggestion
The official book club resource page for WWCode-London 📚
```

File: resources/bookclub/README.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+<td>
+Crucial Conversations by Joseph Grenny & Kerry Patterson              
+<td>
+Crucial Conversations: Tools for Talking When Stakes are High discusses how to handle disagreements and high-stakes communication. It is written on the premise that when you are stuck in any situation – whether it’s at home or work – there is a crucial conversation keeping you from accomplishing the desired results. If you can learn to speak up in these crucial moments effectively, then you can accomplish the results you are after.
+<td>
+<tr>
+
+<tr>
+<td><b>
+  <img src="https://images-na.ssl-images-amazon.com/images/S/compressed.photo.goodreads.com/books/1488319495i/22085568.jpg" />
```
Comment (dricazenck) [inline resources/bookclub/README.md:60]: this image does not render in the page, I am not sure why

File: resources/bookclub/README.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+<tr>
+
+</tbody>
+</table>
+
+&nbsp;
+&nbsp;
+&nbsp;
+
+## Social
```
Comment (dricazenck) [inline resources/bookclub/README.md:112]: considering this page is an internal resource, I don't think is necessary to add Social. Maybe only if you add link to the slack channel for this program.

File: resources/bookclub/README.md (hunk 1 above)
Comment (j4sum) [reply resources/bookclub/README.md]: Changed it, thanks.

File: resources/bookclub/README.md (hunk 2 above)
Comment (j4sum) [reply resources/bookclub/README.md]: Removed and added a new link.

File: resources/bookclub/README.md (hunk 3 above)
Comment (j4sum) [reply resources/bookclub/README.md]: Added the slack and meetup links only.

**Review by bharitwal — APPROVED** [review]

**Pull request merged**